pkg/engine/lifeycletest/test_plan.go

```diff
@@ -210,7 +210,8 @@ func (p *TestPlan) Run(t *testing.T, snapshot *deploy.Snapshot) *deploy.Snapshot
 		if !step.SkipPreview {
 			previewSnap := CloneSnapshot(t, snap)
 			previewTarget := p.GetTarget(previewSnap)
-			_, res := step.Op.Run(project, previewTarget, p.Options, true, p.BackendClient, step.Validate)
+			// Don't run validate on the preview step
+			_, res := step.Op.Run(project, previewTarget, p.Options, true, p.BackendClient, nil)
```

Comment:
Wondering if we should add a `dryRun`/`preview` flag to the validator and then allow it to hit the preview here?

Reply:
Could do, but nothing would use it right now. Would just result in a lot of `if dryRun { return }` lines added to the current validator set.